EBA/Taschenrechner/Geometrie/Geometrie.cs

```diff
@@ -12,7 +12,7 @@ namespace Taschenrechner
 {
     public partial class GeometrieForm : Form
     {
-        public int helper = 0;
+        public double helper = 0;
```

Comment:
can we change the name of helper to? And can you put a comment on this public variable?